first_campaign/electronic-door-lock-102.txt

```diff
@@ -0,0 +1,38 @@
+[name] Top solution Cost
```

Comment:
This record can be named

Reply:
Just not named correctly locally and covered the old name without inserting mine